tests/src/lv_test_conf_sdl.h

```diff
@@ -0,0 +1,3 @@
+#define LV_USE_DRAW_SDL 1
```

Comment:
I'd rather define these in [lv_test_conf.h](https://github.com/lvgl/lvgl/pull/6505/files#diff-e282720b51ca36e99b2b086a97641a2df42d4c3793485cfeb0e5975e06dbdd51R78).

Reply:
done